ModelBundle/Document/ContentType.php

```diff
@@ -211,7 +211,15 @@ public function setDefaultListable(array $defaultListable)
      */
     public function getName($language = 'en')
     {
-        return $this->names->get($language)->getValue();
+        if ($this->names->containsKey($language)) {
+            return $this->names->get($language)->getValue();
+        }
+
+        if ($this->names->count() > 0) {
+            return $this->names->first()->getValue();
+        }
+
+        return 'open_orchestra_model.content.no_name';
```

Comment:
i think is not a good idea to return a string, i think it's better to return null in this case and it's the object who call this method which test if method return null and return a specific string